README.md

```diff
@@ -174,9 +174,14 @@ To publish Hibernate Reactive to your local Maven repository, run:
 ### Running tests
 
 To run the tests, you'll need to get the test databases running on your 
-machine. There are three ways to start the test databases. 
+machine. There are three ways to start the test databases.
 
-#### If you have Docker installed
+By default (almost) all of the tests will run with PostgreSQL. To use a different database
+such as DB2, you can specify `-Pdb=<DBType>`, for example:
```

Comment:
By the way, why Postgres is the one that has multiple valid values for DBType?

Reply:
If you specify an invalid value for DBType, you will get an error telling you what the valid values are. I thought it was best to keep that list in one place (`DatabaseConfiguration`) and let it self-document instead of duplicating the lists and risking them getting out of sync.

Postgres allows multiple valid values because the Vertx URL itself allows multiple values (AFAIK). Mainly I was just trying to catch all of the main terms that people use to refer to this DB type.

Reply:
I think we can just pick one.  We don't have to do what Vert.x does. As long as there is a list of valid options somewhere. It's also easier for use to code and test.

Reply:
As a trade off we could add a link to the source code with enums.

Reply:
Nah you're right, a hardcoded list is less clicks for the user. Added a hard-coded list of options